pandas/core/generic.py

```diff
@@ -5164,6 +5164,11 @@ def _is_datelike_mixed_type(self):
         f = lambda: self._data.is_datelike_mixed_type
         return self._protect_consolidate(f)
 
+    @property
```

Comment:
you shouldn't need this as ``_is_datelike_mixed_type`` should cover this case, no?

Reply:
`_is_datelike_mixed_type` is not specific to tz-aware blocks nor checks the actual timezone which we need here